### PR TITLE
Loop exercises

### PR DESCRIPTION
**Pull request by jimflores5**
What are the team thoughts about:
a) Comparing the ``while`` practice in the exercises vs. the studio, I see a significant difference in time and effort required.  Modifications?
b) What do you think about switching exercise #4 (launching a rocket) with studio part C (user specifying number of meals to assemble)?  
c) Too many ``for`` problems in the exercises?  Opinions, please.
d) General styling, phrasing, spelling, etc. corrections.

### CODE REVIEW

File: src/chapters/loops/exercises.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+``while`` Practice
+-------------------
+
+Control+click (or right-click) to: `Repl.it here <https://repl.it/@launchcode/WhileLoopExercises>`__
+
+Define three variables for the LaunchCode shuttle - one for the starting
+fuel level, another for the number of astronauts aboard, and the third for
+the altitude the shuttle reaches.
+
+4. Construct ``while`` loops to do the following:
```
Comment (gildedgardenia) [inline src/chapters/loops/exercises.rst]: I think these parts should be clearer. Are you wanting the student to write a loop that continues going until the user enters a valid number?

File: src/chapters/loops/exercises.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    | a. Query the user for the starting fuel level.  Validate that the user
+        enters a positive, integer value greater than 5000 but less than 30000.
+    | b. Use a second loop to query the user for the number of astronauts
+        (up to a maximum of 7). Validate the entry.
+    | c. Use a final loop to monitor the fuel status and the altitude of the
+        shuttle.  Each iteration, decrease the fuel level by 100 units for each
+        astronaut aboard.  Also, increase the altitude by 500 kilometers.
+
+#. Output the result with the phrase, “The shuttle gained an altitude of ___ km.”
```
Comment (gildedgardenia) [inline src/chapters/loops/exercises.rst]: Is this supposed to be a loop question?

File: src/chapters/loops/studio.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    | Vegetable options: [‘peas’, ‘green beans’, ‘edamame’, ‘broccoli’,
+        ‘asparagus’, ‘kale’]
+    | Beverage options: [‘juice’, ‘milk’, ‘soy milk’, ‘soda’, ‘tea’, ‘water’]
+    | Dessert options: [‘apple’, ‘banana’, ‘ice cream’, ‘chocolate’, ‘kiwi’,
+        ‘more kale’]
+
+    Control+click (or right-click) to: `Code here <https://repl.it/@launchcode/LoopstudiopartsAandC>`__
+
+#. Construct a ``for`` loop that assembles a meal for each of 6 astronauts.
+    a. The meals must include one item from each of the source arrays.
```
Comment (gildedgardenia) [inline src/chapters/loops/studio.rst]: Are these items to be selected at random?

File: src/chapters/loops/studio.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+.. image:: figures/array-of-arrays.png
+    :height: 300px
+
+Part B (Self-destruct system)
+-----------------------------
+
+If the shuttle gets hijacked by space pirates, the astronauts can activate
+a self-destruct sequence to provide some drama for the viewers at home.
+
+In order to prevent a rogue ‘naut from activating the code, it takes **two**
```
Comment (gildedgardenia) [inline src/chapters/loops/studio.rst]: I recommend spelling out astronaut.

File: src/chapters/loops/exercises.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
+Loop Exercises
+==============
+
+| Practice makes better.  Repetition is a good thing.
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/loops/exercises.rst]: I agree - the joke is funny.  I also added a code sample and some more text to help reinforce the idea of how loops benefit us.

File: src/chapters/loops/exercises.rst [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+-----------------
+
+Control+click (or right-click) to: `Repl.it here <https://repl.it/@launchcode/ForLoopExercises>`__
+
+#. Construct ``for`` loops that accomplish the following tasks:
+    | a. Print the numbers 0 - 20, one number per line.
+    | b. Print only the ODD values from 3 - 29, one number per line.
+    | c. Print the EVEN numbers 12 to -14 in descending order, one number
+        per line.
+    | d. **Challenge** - Print the numbers 50 - 20 in descending order, but only
```
[the remark this answers is not in the document]
Comment (jimflores5) [reply src/chapters/loops/exercises.rst]: I think we agreed to only have bonus tasks with the studios.  I replaced "Challenge" with a more specific hint for this task.

File: src/chapters/loops/studio.rst (hunk 3 above)
Comment (jimflores5) [reply src/chapters/loops/studio.rst]: The students have not yet learned how to use the Math module, so the meal assembly will just iterate sequentially through the arrays.  (That's why they are all the same length).

File: src/chapters/loops/exercises.rst (hunk 1 above)
Comment (jimflores5) [reply src/chapters/loops/exercises.rst]: I see your point.  I updated the wording to try to clarify the tasks.

File: src/chapters/loops/exercises.rst (hunk 2 above)
Comment (jimflores5) [reply src/chapters/loops/exercises.rst]: Nope.  I will re-word to make this clear.

**Pull request merged**